docs/source/algorithms/PelicanReduction-v1.rst

```diff
@@ -0,0 +1,35 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+Reduces the data from the Pelican instrument from time of flight versus two theta to S(Q,w). Data can be corrected for background with an empty can subtraction and normalised to vanadium (both optional). Output can be for a powder S(Q,w) or for a single crystal S(Q,w) in the latter case an nxspe file is written. Data can be processed for the specified region of Q and w space. Data can also be processed for the lamda/2 option for Pelican. As a default the output for a powder also includes the Q integrated data S(ω) and the energy integrated data S(Q).
+
+The portion of the detector that is used is defined in the config file. A frame overlap option is also available, where for low temperature data, a portion of the neutron energy gain spectrum can be used to extend the range of the neutron energy loss side.
+
+Usage
+-----
+
+.. testcode:: PelicanReductionExample
+
+    test = PelicanReduction('44464', EnergyTransfer='-2,0.05,2', MomentumTransfer='0,0.05,2', ConfigurationFile='pelican_doctest.ini')
```

Comment:
This doc test is good :+1: , could we provide some system tests to validate the algorithm? People typically don't run doc tests to validate the output (and it wouldn't show an error cropping up)

This PR needs system tests before I would personally be comfortable approving it. As we would have no automated way to know if the data produced is still valid, say if something indirectly changed and altered the data in the Mantid framework

A couple of data files (if possible), tests and reference outputs of the various branches (such as separate tests for SofQW and NXSPE) helps prevent subtle changes to the produced data, especially long-term where things might change in the underlying algorithms

Reply:
Added two system tests to handle the SOFQW and NXSPE cases.

On a different issue, the tests failed on two areas completely unrelated to changes here. Could you kick it off again as I don't have the authorization.

Finally, where do I add the release comments about the addition of the Pelican reduction functionality?